scripts/cluster_network.py

```diff
@@ -611,6 +605,19 @@ def update_bus_coordinates(
     mode = params.mode
     solver_name = snakemake.config["solving"]["solver"]["name"]
 
+    simplified_busmap = pd.read_csv(
+        snakemake.input.simplified_busmap, index_col=0
+    ).squeeze()
+    if isinstance(simplified_busmap, pd.DataFrame):
+        simplified_busmap = simplified_busmap.iloc[:, 0]
```

Comment:
Is there any case where simplified busmap can be a `pd.DataFrame`? If not, can remove this check.